### PR TITLE
Added plugins support

### PR DESCRIPTION
**Pull request by tejash-jl**
Now you can pass the extra parameter for the plugins, like the serverUrl for upload etc.
https://alex-d.github.io/Trumbowyg/documentation/plugins/#plugin-upload

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +22,23 @@
   "dependencies": {
     "react": "^15.3.2",
     "react-dom": "^15.3.2",
-    "trumbowyg": "^2.3.0"
+    "trumbowyg": "2.12.0"
```
Comment (Alex-D) [inline package.json]: Please let the `^`

File: package.json [hunk 2]
```diff
@@ -21,17 +22,23 @@
...
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
     "clean-webpack-plugin": "^0.1.13",
     "copy-webpack-plugin": "^4.0.0",
     "eslint": "^3.9.1",
     "eslint-loader": "^1.6.0",
     "eslint-plugin-react": "^6.4.1",
     "file-loader": "^0.9.0",
+    "react-hot-loader": "^4.8.0",
```
Comment (Alex-D) [inline package.json]: Why all these new deps are needed?

**Review by Alex-D — CHANGES_REQUESTED** [review]

**Comment by tejash-jl** [comment]
I have removed the changes suggested. Please check.

**Review by Alex-D — APPROVED** [review]
LGTM :)

@sochix can you merge (squash) that?

**Comment by sochix** [comment]
Thanks everyone! Merged!

**Pull request merged**